JeMPI_Apps/JeMPI_Shared_Source/custom/CustomLibMPIConstants.java

```diff
@@ -41,6 +41,22 @@ query goldenRecordByUid($uid: string) {
             GoldenRecord.city
             GoldenRecord.phone_number
             GoldenRecord.national_id
+            GoldenRecord.entity_list @facets(score) {
```

Comment:
I think there's a Scala file that you need to update as well, it's responsible for generating this file.

Reply:
will check that out :+1: